joomla-projects/privacy-framework
# Changed the backend icon - com_actionlogs

### Summary of Changes
Just a cosmetic change.
Changed the "icon" of com_actionlogs administration from the generic icon to a more meaningful "list".

### Testing Instructions
Nothing special. Navigate in the backend of the com_actionlogs and see the list icon instead of the generic one.

### Expected result
![immagine](https://user-images.githubusercontent.com/733133/42685026-295c9e24-8692-11e8-9d5f-481a59e0dc2b.png)


### Actual result
![immagine](https://user-images.githubusercontent.com/733133/42685039-362592a0-8692-11e8-9a69-a86c0f3c0250.png)



### Documentation Changes Required
None
